calico/getting-started/kubernetes/vpp/openshift.mdx

```diff
@@ -0,0 +1,211 @@
+---
+description: Install Calico VPP on an OpenShift 4 cluster.
+---
+
+# Install an OpenShift 4 cluster with Calico VPP
+
+## Big picture
+
+Install an OpenShift 4 cluster with {{prodname}} VPP.
+
+## Value
+
+Augments the applicable steps in the [OpenShift documentation](https://cloud.redhat.com/openshift/install)
+to install {{prodname}}.
+
+## How to
+
+### Before you begin
+
+- Ensure that your environment meets the {{prodname}} [system requirements](../openshift/requirements.mdx).
+
+- **If installing on AWS**, ensure that you have [configured an AWS account](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-account.html) appropriate for OpenShift 4,
+  and have [set up your AWS credentials](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html).
+  Note that the OpenShift installer supports a subset of [AWS regions](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-account.html#installation-aws-regions_installing-aws-account).
+
+- Ensure that you have a [RedHat account](https://cloud.redhat.com/). A RedHat account is required to obtain the pull secret necessary to provision an OpenShift cluster.
+
+- Ensure that you have installed the OpenShift installer **v4.13 or later** and OpenShift command line interface from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
+
+- Ensure that you have [generated a local SSH private key](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-default.html#ssh-agent-using_installing-aws-default) and have added it to your ssh-agent
+
+### Create a configuration file for the OpenShift installer
+
+Create a staging directory for the installation. This directory will contain the configuration file, along with cluster state files, that OpenShift installer will create:
+
+```
+mkdir openshift-tigera-install && cd openshift-tigera-install
+```
+
+Run OpenShift installer to create a default configuration file:
+
+```
+openshift-install create install-config
+```
+
+:::note
+
+Refer to the [OpenShift installer documentation](https://cloud.redhat.com/openshift/install) for more information
+about the installer and any configuration changes required for your platform.
+
+:::
+
+Once the installer has finished, your staging directory will contain the configuration file `install-config.yaml`.
+
+### Update the configuration file to use {{prodname}}
+
+Override the OpenShift networking to use Calico and update the AWS instance types to meet the [system requirements](../openshift/requirements.mdx):
```

Comment:
Here we have an AWS-specific configuration. This either needs to be called out separately or we need to label the whole doc as an AWS install.

Reply:
See 1st comment.